meta-refkit/classes/refkit-image.bbclass

```diff
@@ -174,6 +174,10 @@ IMAGE_INSTALL += "util-linux"
 export ALTERNATIVE_PRIORITY_SHADOW ?= "305"
 IMAGE_INSTALL += "shadow"
 
+# Install thermal management daemon by default for
+# handling platform thermal management.
+IMAGE_INSTALL += "thermald"
```

Comment:
Are there board specific configs? Why are such needed? I would assume it would work similar way as cpufreq. If you think about normal Linux distros for example there's no way those can know about all possible motherboard/cpu/cooling combinations. Thermal limit points can be read from hardware as well as fan info. So similar thermal profiles can be used as BIOS do (silent, normal, cool).